packages/utils/node-resolver-core/src/NodeResolver.js

```diff
@@ -429,11 +430,16 @@ export default class NodeResolver {
         let moduleDir = path.join(dir, 'node_modules', moduleName);
         let stats = await this.fs.stat(moduleDir);
         if (stats.isDirectory()) {
+          const realModuleDir = this.fs.realpathSync(moduleDir);
```

Comment:
Can we avoid sync fs calls and use the async version instead?

Reply:
```suggestion
          const realModuleDir = await this.fs.realpath(moduleDir);
```